instagram-app/src/App.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PostPage from './components/PostContainer/PostPage';
```

Comment:
Watch the names of the files in your imports. Make sure that they match the files in your project. 👍